go/internal/initutil/ipfs.go

```diff
@@ -171,7 +172,7 @@ func (m *Manager) getLocalIPFS() (ipfsutil.ExtendedCoreAPI, *ipfs_core.IpfsNode,
 
 			// Setup MC
 			if m.Node.Preset != PresetAnonymity {
-				mcOpt := libp2p.Transport(mc.ProximityTransportConstructor(m.ctx, logger))
+				mcOpt := libp2p.Transport(proximity.NewTransport(m.ctx, logger, mc.NewDriver(logger)))
```

Comment:
Enable your driver by default in initutil

Feel free to add a new flag, i.e. `--without-mcconnectivity`

Reply:
I added a new flag `MultipeerConnectivity` like `LocalDiscovery`.